src/typeMapper.js

```diff
@@ -83,7 +83,8 @@ export function toGraphQL(sequelizeType, sequelizeTypes) {
       sequelizeType instanceof DATEONLY ||
       sequelizeType instanceof TIME ||
       sequelizeType instanceof BIGINT ||
-      sequelizeType instanceof DECIMAL) {
+      sequelizeType instanceof DECIMAL ||
+      ['NCHAR', 'SMALLDATETIME'].indexOf(sequelizeType) > -1) {
```

Comment:
defined as strings in your sequelize model?

Reply:
Sequelize does not handle these types and therefore `sequelizeType` is just a string in this case. This thing needs to be handled in Sequelize. Closing this PR.

Reply:
But do need to fix this issue in mentioned in comment: https://github.com/mickhansen/graphql-sequelize/issues/611#issuecomment-439342438